book/index.qmd

```diff
@@ -1,30 +1,31 @@
 # Preamble {.unnumbered}
 {{< include _setup.qmd >}}
 
-```{r C0 seed}
+```{r C0 seed, echo = FALSE}
 set.seed(0)
 ```
 
-Welcome to the Machine Learning in R 3 universe (mlr3verse), let us show you some of its magic.
-Before we begin, make sure you have installed `r mlr3` if you want to follow along, we recommend installing the full universe at once:
+Welcome to the Machine Learning in R universe (mlr3verse), let us show you some of its magic!
+Before we begin, make sure you have installed `r mlr3` if you want to follow along.
+We recommend installing the complete `r mlr3verse`, which will install all of the important packages.
 
 ```{r C0 install mlr3verse, eval = FALSE}
 install.packages("mlr3verse")
 ```
 
-You can also just install the base package:
+Or you can install just the base package:
 
 ```{r C0 install mlr3, eval = FALSE}
 install.packages("mlr3")
 ```
 
-In this first example we'll show you the most basic use-case, train and predict.
+In our first example, we will show you some of the most basic functionality -- training a model and making predictions.
 
 ```{r C0 egBasic}
-library("mlr3")
+library(mlr3)
```

Comment:
Happy with this change but need to make sure we do this everywhere -- I've been changing as I see them